docs/tools/cicd/integration_test_failures.md

```diff
@@ -0,0 +1,51 @@
+# Integration Test Failures
+
+There are two (2) to receive notifications of integration test failures. 
+One is in [MilMove CircleCI dashboard](https://app.circleci.com/pipelines/github/transcom/mymove) 
+and the other is from the [GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) page.
+
+## CircleCI
+
+You will see a `Failed` notification on the dashboard page. Click on the `Failed` notification/label to see what step in the CI/CD has failed.
+![Screen Shot 2021-11-09 at 2 38 48 PM](https://user-images.githubusercontent.com/1359520/141003337-a15e2cb5-2036-4752-af00-85b836dd5a44.png)
+
+After you click on the `Failed` notification/label, you will be presented with a list of actions that were taken or are still in progress for the commit.
+Click on the failed indicator to see the details of what is failing.
+![Screen Shot 2021-11-09 at 2 38 56 PM](https://user-images.githubusercontent.com/1359520/141003369-e26facb8-7f45-4794-acfb-a70f107b879f.png)
+
+These details will show the terminal output that you would see if the tests were running locally.
+<img width="1028" alt="Screen Shot 2021-11-09 at 3 58 47 PM" src="https://user-images.githubusercontent.com/1359520/141011976-2dada386-b769-4d45-aa4f-24c18c87170e.png">
+
+
+Once you see these logs, you are able to look at the error messages and determine next steps to fix the failed test(s).
+
+### Re-run 
+Pushing a new commit to your branch will automatically kick off CI/CD pipeline. However, if there are no changes and you would like to re-run 
+the tests you can click on `Rerun` in the upper right-hand corner of the CircleCI page.
+![Screen Shot 2021-11-09 at 3 05 03 PM](https://user-images.githubusercontent.com/1359520/141004542-1ea6ff51-1312-4705-8a4b-1a97c06b5fe8.png)
+
+Unless you know you want to re-run the entire pipeline select `Rerun` > `Rerun Workflow from Failed` is good enough. This option only re-runs
+the failed tests.
+![Screen Shot 2021-11-09 at 3 05 13 PM](https://user-images.githubusercontent.com/1359520/141004562-3c36af65-0851-45b7-a302-450997cfd07e.png)
+
+
+## GitHub pull request
```

Comment:
I think we should get rid of images in this section and [point to the documentation that exists for GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks).

Reply:
@rogeruiz  I can point to that but it actually doesn't describe what I have described. I agree with reusing documentation but also I want to make things clear. I don't get the same information from that GH  page.